code/game/turfs/turf.dm

```diff
@@ -183,6 +183,8 @@
 	var/old_ex_id = explosion_id
 
 	Destroy()	//❄
+	if(initialized)
+		world.log << "Fuck your shit ass cunt"
```

Comment:
This confused me a lot when I woke up and someone said there was a bunch of "Fuck your shit ass cunt" in the runtime logs.